### PR TITLE
Jurstify

### PR DESCRIPTION
**Pull request by theQuazz**
There was a decent amount of work involved in converting Jass to Jurst, mostly adding a line like this to the top of our files. I don't know if it is worthwhile to read through this, I think testing it out is probably more efficient.

```
library LibraryName initializer onInit requires ID
```

### CODE REVIEW

File: wurst/systems/units/hostiles/trolls/Assassin.wurst [hunk 1]
```diff
@@ -1,4 +1,5 @@
 package Assassin
+import ID
```
Comment (jonathanmclaus) [inline wurst/systems/units/hostiles/trolls/Assassin.wurst]: Why did you add this to all entity classes? Seems like it isn't being used.

Comment (theQuazz) [reply wurst/systems/units/hostiles/trolls/Assassin.wurst]: @jonathanmclaus it is being used. You probably have the old war3map.j.
